.eslintrc.js

```diff
@@ -65,6 +65,7 @@ module.exports = {
     "@typescript-eslint/no-shadow": "error",
     '@typescript-eslint/prefer-for-of': 'error',
     '@typescript-eslint/prefer-function-type': 'error',
+    '@typescript-eslint/return-await': 'error',
```

Comment:
The [How to Use](https://typescript-eslint.io/rules/return-await/#how-to-use) section in TS ESlint docs says you need to also set `"no-return-await": "off"` to avoid errors.